app/views/content/get-help-hiring/find-the-best-teachers.md

```diff
@@ -0,0 +1,19 @@
+---
+order: 1
+title: Find the best teachers
+---
+## Include the school perks
+List all the perks you can have working at your school
+
+## Another subtitle
+Bla bla bla
```

Comment:
I'm a little wary of this going live, even if it's not linked from anywhere yet.  Should we rename the files to something random so at least we're not associating temp content with possibly real future URLs?

Reply:
Hey mate, we're going to remove this file before the branch is merged. This is only there so we can see the page in the review app and locally.